_includes/aside/toc.html

```diff
@@ -4,5 +4,7 @@
     </div>
     <div class="contribute-box">
         <p><a href="../../contribute" class="contribute-button">Contribute an Article</a></p>
+        <br>
```

Comment:
Here it takes the user to the newsletter so this button not required. On top anyway we have newsletter option in menu level.